docs/05-technical-reference/00-architecture/evnt-01-architecture.md

```diff
@@ -2,35 +2,27 @@
 title: Eventing Architecture
 ---
 
-Eventing uses NATS to implement the Event Publisher Proxy and the Eventing Controller which work together to process and deliver events in Kyma. See how Eventing works inside a Kyma cluster:
-
-![Eventing architecture](./assets/evnt-architecture.svg)
-
-1. Kyma user creates the Application custom resource (CR).
-2. Application Operator watches for Application CRs and creates a HTTPSource CR.
-3. EventSource Controller watches for HTTPSource CRs and deploys the HTTP Source Adapter.
-4. Application Broker watches for Application CRs, exposes events, deploys NATS subscriptions, and labels secrets in the User Namespace.
-5. Kyma user creates a [Subscription CR](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) for an event.
+Eventing uses NATS to implement the Event Publisher Proxy and the Eventing Controller, which work together to process and deliver events in Kyma. See also the [Event processing and delivery](../evnt-01-event-processing.md) document for more information.
```

Comment:
Eventing uses NATS as a default eventing backend. The event publisher proxy and the eventing controller are just the two components that we built to connect to this backend